Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -367,17 +367,11 @@ - (void)loadAdViewAdForParameters:(id<MAAdapterResponseParameters>)parameters ad
     }
     else
     {
-        // Check if adaptive ad view sizes should be used
-        BOOL isAdaptiveAdViewEnabled = [parameters.serverParameters al_boolForKey: @"adaptive_banner"];
-        if ( isAdaptiveAdViewEnabled && ALSdk.versionCode < 13020099 )
-        {
-            [self userError: @"Please update AppLovin MAX SDK to version 13.2.0 or higher in order to use Vungle adaptive ads"];
-            isAdaptiveAdViewEnabled = NO;
-        }
```

Comment:
removed this code. cause this is not working as expected.

If I set this `[self.adView setExtraParameterForKey: @"adaptive_banner" value: @"true”];` in my test app then I have to use older api `[[MAAdView alloc] initWithAdUnitIdentifier:adUnit]` so that adapter can get the param @“adaptive_banner”  from `[parameters.serverParameters al_boolForKey: @"adaptive_banner”]`. 

If I use the new API [[MAAdView alloc] initWithAdUnitIdentifier: adUnit configuration: config];. Then adapter cannot get the param @“adaptive_banner”  from [parameters.serverParameters al_boolForKey: @"adaptive_banner”].